views/index.pug

```diff
@@ -0,0 +1,84 @@
+html
+head
+    link(rel="stylesheet" href="/styles.css")
+    script(src="../other.js")
+    body
+     h1.title.headersCentered Welcome to Find a Cookie
+     h2.headersCentered Narrow down your cookie search or add a cookie recipe.
+    ul.list 
+      p Choose one of the following search flows:
+      li.listItems 
+        p 
+          strong GET 
+          | /allCookies
+        div#getAll
+            p Method: GET 
+            p Route: http://localhost:1990/cookies/allCookies 
```

Comment:
Adding `allCookies` to this route feels redundant. A GET to `/cookies` returning all cookies makes sense.